libraries/rush-lib/src/logic/pnpm/PnpmShrinkwrapFile.ts

```diff
@@ -346,7 +346,8 @@ export class PnpmShrinkwrapFile extends BaseShrinkwrapFile {
   private _disallowInsecureSha1(
```

Comment:
Seems like there should be other errors that should also mention the subspace.

Reply:
@L-Qun - do you want to add the subspace name to other errors in this PR, or do that in follow-ups?